bindings/cpp/AndroidSecureStorage/app/src/main/java/com/cronos/play/SecureStorage.kt

```diff
@@ -0,0 +1,71 @@
+package com.cronos.play
+
+import android.content.Context
+import androidx.security.crypto.EncryptedFile
+import androidx.security.crypto.MasterKey
+import androidx.security.crypto.MasterKey.Builder
+import java.io.File
+
+class SecureStorage {
```

Comment:
I assume many of those functions (like file opening etc.) may throw Java exceptions -- how will the native code know the exception happened? should the functions have try-catch and capture the exception into the return value?

Reply:
i'll add exception handling code